### DISCUSSION TITLE
How was wind_data.png created?

### DISCUSSION THREAD

**Discussion by saintsjd** [kamzek/deck.gl-particle#9] [Q&A] [answered]
How was `wind_data.png` was created? 

I am having difficulty using deck.gl/particle because I am not sure how to process my GFS data into the correct format. 

Perhaps we can discuss here how to properly format png files, then I can update the documentation to help developers format their data for use with the package.

**Comment by saintsjd** [comment]
We could also provide an example of how weatherlayers provides this imagery for you.

**Comment by zakjan** [comment, marked as answer]
Hi, thanks for noticing, I have added the instructions to the readme. Does it help you?

```
gdalbuildvrt -separate wind_data.vrt u.grib v.grib
gdal_translate -ot Byte -scale -128 127 0 255 wind_data.vrt wind_data.png
```
